caboose.go

```diff
@@ -65,6 +68,10 @@ type Caboose struct {
 }
 
 func NewCaboose(config *Config) (ipfsblockstore.Blockstore, error) {
+	if config.SaturnLoggerJWT == "" || len(config.SaturnLoggerJWT) == 0 {
+		return nil, errors.New("JWT token required for Saturn Logger")
+	}
```

Comment:
:warning:  this breaks flow for local development OR requires everyone on the team to get their own token.
iiurc last time we talked about this @guanzo  noted it is ok to disable sending logs when JWT token is not set.

Reply:
Done.